index.html

```diff
@@ -8,7 +8,7 @@
 </head>
 <body>
   <div id="root"></div>
-  <script></script>
   <script src="dist/bundle.js"></script>
```

Comment:
`/dist` in the URL is fine. The server maps it to the local `public` directory. What we used to be doing was actually installing the files to the `/dist` directory.